src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -122,6 +124,26 @@ where Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Window
 #endif
 			};
 
+			List<string> excludeNamespaces = new List<string>();
+			List<string> includeNamespaces = new List<string>();
+
+#if !HAS_UNO_WINUI
+			// For UWP compilation we need to ignore these namespaces when not explicitly generating
+			// for related projects.
+			excludeNamespaces.Add("Windows.UI.Dispatching");
+			excludeNamespaces.Add("Windows.UI.Composition");
+			if (baseName == "Uno.UI.Dispatching")
+			{
+				excludeNamespaces.Clear();
+				includeNamespaces.Add("Windows.UI.Dispatching");
+			}
+			if (baseName == "Uno.UI.Composition")
+			{
+				excludeNamespaces.Clear();
+				includeNamespaces.Add("Windows.UI.Composition");
+			}
```

Comment:
I feel this is more readable. But it's a minor change so feel free if you don't want to take it :)

```suggestion
			if (baseName == "Uno.UI.Dispatching")
			{
				includeNamespaces.Add("Windows.UI.Dispatching");
			}
			else if (baseName == "Uno.UI.Composition")
			{
				includeNamespaces.Add("Windows.UI.Composition");
			}
			else
			{
				excludeNamespaces.Add("Windows.UI.Dispatching");
				excludeNamespaces.Add("Windows.UI.Composition");
			}
```

Reply:
Yes, that's probably nicer, amended to last commit 👍